Cargo.toml

```diff
@@ -10,14 +10,21 @@ repository = "https://github.com/prometheus/client_rust"
 homepage = "https://github.com/prometheus/client_rust"
 documentation = "https://docs.rs/prometheus-client"
 
+[features]
+protobuf = ["dep:prost", "dep:prost-types", "dep:prost-build", "dep:prometheus-client-derive-proto-encode", "dep:void"]
+
 [workspace]
-members = ["derive-text-encode"]
+members = ["derive-text-encode", "derive-proto-encode"]
 
 [dependencies]
 dtoa = "1.0"
 itoa = "1.0"
 parking_lot = "0.12"
+prometheus-client-derive-proto-encode = { version = "0.1.0", path = "derive-proto-encode", optional = true }
```

Comment:
Exploring the thought on whether we could use the same derive macro for both text and protobuf. I guess we can not feature flag the generated protobuf code via `#[cfg(feature = "protobuf")]` as we would not have access to the flag in the downstream user crate?

Reply:
That is cool if we use the same derive macro, I'll do some experiments. 🙂

Reply:
I investigated but yeah, I couldn't find a way to do that.